src/pages/guidelines/frameworks.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Frameworks
+description: The Carbon for IBM.com React and Web components libraries provide front-end developers and engineers a collection of reusable components to build websites and user interfaces. Adopting the libraries enable developers to use consistent markup, styles, and behavior in prototype and production work.  
+---
+
+<PageDescription>
+
+The Carbon for IBM.com React and Web components libraries provide front-end developers and engineers a collection of reusable components to build websites and user interfaces. Adopting the libraries enable developers to use consistent markup, styles, and behavior in prototype and production work.
+</PageDescription>
+
+
+<AnchorLinks>
+
+<AnchorLink>What is React?</AnchorLink>
+<AnchorLink>What is Web components?</AnchorLink>
+<AnchorLink>Why Carbon for IBM.com only support React and Web components?
+</AnchorLink>
+<AnchorLink>Resources</AnchorLink>
+
+</AnchorLinks>
+
+## What is React?
+
+React, sometimes calls React.js, is a JavaScript library and a tool for building UI components. It allows developers to create components fast, scalable, and is simple to use. 
+
+Below are a few websites, documents, and tutorial links to help you learn more about React.
+
+* Name of the link (URL)
+* Name of the link (URL)
+* Name of the link (URL)
```

Comment:
Sorry, looks like I approved the wrong pull request while I was going in and out of pull requests too fast. My bad. I know this is a work in progress. I'm adding a request change to something I see will definitely change to avoid any accidental merges.

Reply:
No worry, @photodow . This task will be worked on by Jeff in Sprint 21-12. Once Jeff is ready, I will share the preview with designers to take a look and share feedback - FYI.